src/components/POLine/Item/MaterialTypesForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Field, FieldArray } from 'redux-form';
+import { Row, Col, Button, TextField } from '@folio/stripes/components';
+
+class MaterialTypesForm extends Component {
+  constructor(props) {
+    super(props);
+    this.renderForm = this.renderForm.bind(this);
+    this.renderSubForm = this.renderSubForm.bind(this);
+  }
+
+  renderForm = ({ fields }) => {
+    return (
+      <Row>
+        <Col xs={12}>
+          {fields.length === 0 &&
+            <Col xs={12}>
+              <div><em>- Please Add Material Types -</em></div>
+            </Col>
+          }
+          {fields.map(this.renderSubForm)}
+        </Col>
+        <Col xs={12} style={{ paddingTop: '10px' }}>
+          <Button onClick={() => fields.push('')}>+ Add Material Type</Button>
```

Comment:
The same about hardcoded strings here.
Also, as far as I know, it was decided to not use plus signs in buttons. @cherewaty, am I right?